### PR TITLE
Docker socket doesn't have to be writeable to resolve container names

### PR DESCRIPTION
**Pull request by tobiasmuehl**
##### Summary
Documentation tells users to bind docker socket writeable, although it doesn't seem necessary for docker container name resolution. 

##### Component Name
Gitbook

##### Additional Information
Other users reporting success with this method can be found in this [issue thread](https://github.com/netdata/netdata/issues/3972#issuecomment-419721436)

### CODE REVIEW

File: packaging/docker/README.md [hunk 1]
```diff
@@ -60,29 +60,14 @@ If you don't want to use the apps.plugin functionality, you can remove the mount
 
 ### Docker container names resolution
 
-If you want to have your container names resolved by netdata, you need to do two things:
-1) Make netdata user be part of the group that owns the socket.
+If you want to have your container names resolved by netdata, make netdata user be part of the group that owns the socket.
    To achieve that just add environment variable `PGID=[GROUP NUMBER]` to the netdata container,
```
Comment (joelhans) [inline packaging/docker/README.md]: This shouldn't be indented any more.

File: packaging/docker/README.md [hunk 2]
```diff
@@ -60,29 +60,14 @@ If you don't want to use the apps.plugin functionality, you can remove the mount
 
 ### Docker container names resolution
 
-If you want to have your container names resolved by netdata, you need to do two things:
-1) Make netdata user be part of the group that owns the socket.
+If you want to have your container names resolved by netdata, make netdata user be part of the group that owns the socket.
```
Comment (joelhans) [inline packaging/docker/README.md]: Can you change this sentence:

```suggestion
If you want to have your container names resolved by Netdata, make the `netdata` user be part of the group that owns the socket.
```

File: packaging/docker/README.md (hunk 1 above)
Comment (joelhans) [reply packaging/docker/README.md]: Also, please capitalize Netdata: 

```suggestion
   To achieve that just add environment variable `PGID=[GROUP NUMBER]` to the Netdata container,
```

I know there's a lot of inconsistency in the docs already but every change helps!

**Comment by joelhans** [comment]
Also, let's consider how this connects to #6441. Working on the same part of the same file here.

**Comment by ViViDboarder** [comment]
> Also, let's consider how this connects to #6441. Working on the same part of the same file here.

Yea... My diff takes out the GID logic in the run script.

I had originally added some logic to just update the `netdata` UID and GID, but I ran into some issues. I can test this and try to add it back in.

**Comment by ViViDboarder** [comment]
Ok. Got something workable based on what you have suggested. 👍 

I'll try to roll this updated documentation into my diff.

**Comment by cakrit** [comment]
Actually, this information already exists on the current version of the README. Apologies for the delay @tobiasmuehl !!!

**Pull request closed without merging**